test/e2e/multinode/e2e_slash_test.go

```diff
@@ -119,5 +125,54 @@ func slashTest(t *testing.T) {
 	}
 
 	assert.Equal(t, expectedBalance, balance, fmt.Sprintf("The expected balance should be %s", expectedBalance))
+}
+
+// Test disable slash
+func disableSlashTest(t *testing.T) {
+	log.Infoln("===================================================================")
+	log.Infoln("======================== Test disableSlash ===========================")
+
+	setupSlash()
+
+	transactor := tc.NewTestTransactor(
+		t,
+		tc.SgnCLIHomes[0],
+		tc.SgnChainID,
+		tc.SgnNodeURI,
+		tc.ValAccounts[0],
+		tc.SgnPassphrase,
+	)
+
+	setupValidators(t, transactor)
+
+	paramChanges := []govtypes.ParamChange{govtypes.NewParamChange("slash", "EnableSlash", "false")}
+	content := govtypes.NewParameterProposal("Slasg Param Change", "Update EnableSlash", paramChanges)
+	submitProposalmsg := govtypes.NewMsgSubmitProposal(content, sdk.NewInt(1), transactor.Key.GetAddress())
+	transactor.AddTxMsg(submitProposalmsg)
 
+	proposalID := uint64(1)
+	byteVoteOption, _ := govtypes.VoteOptionFromString("Yes")
+	voteMsg := govtypes.NewMsgVote(transactor.Key.GetAddress(), proposalID, byteVoteOption)
+	transactor.AddTxMsg(voteMsg)
+
+	_, err := tc.QueryProposal(transactor.CliCtx, proposalID, govtypes.StatusPassed)
+	require.NoError(t, err, "failed to query proposal 1 with passed status")
+
+	shutdownNode(1)
+
+	nonce := uint64(0)
+	_, err = tc.QueryPenalty(transactor.CliCtx, nonce, 1)
+	assert.Error(t, err, "failed to query penalty 0 with 1 sig")
```

Comment:
the error msg is a bit confusing, as we are expecting an error here. how about something like `get penalty 0 with 1 sig, should be 0 sig`

Reply:
I think it can be `get penalty 0 with 1 sig should fail`

Reply:
sounds good